### DISCUSSION TITLE
init_commands

### DISCUSSION THREAD

**Discussion by jeandavid59** [hassio-addons/addon-ssh#642] [General]
Hello , I would like to define prompt when I open a new terminal window. Actually I use these commands 

setopt PROMPT_SUBST
PROMPT='%{$(pwd|grep --color=always /)%${#PWD}G%} > '

If I add these commands to "init_commands" it doesn't work.
Thanks for your help.
Regards

**Comment by zach6439** [comment]
Did you ever resolve this? I am currently trying to get some aliases to load when the add on restarts.

Ive tried putting this directly into the init_command as well as via a script, and have not been able to get it to work, however if I run it within the terminal, the aliases work until the add on is rebooted.

Init command: `source /config/.shellstuff/.zshrc`

Using /config/.shellstuff/ as the config folder is persistent and the subfolder is just for organizing.

I also tried to run a script that calls that source to see if that work around would work and it also doesn't work in the init command.
I did verify that the script runs in the terminal and does source the appropriate file.

Init command: `/config/.shellstuff/init_command.sh`
Script contents: `source /config/.shellstuff/.zshrc`
